### ISSUE TITLE
Reacting to new group creation requests from the contract

### ISSUE THREAD

**Issue by pdyraga** [keep-network/keep-core#3043]
React to the new group creation requests from the `RandomBeacon` contract and kick off random beacon DKG.

**Comment by mswilkison** [comment]
Consolidated into the random-beacon tracking epic #3044. This isn't independent work — it's blocked on the same missing layer (wiring `pkg/chain/ethereum/beacon.go` to the generated `RandomBeacon` bindings), which is deliberately deferred behind FROST/ROAST GA → a permissionless-participation decision. The current chaosnet stub is fit for the permissioned operator set. Full evidence and the dependency chain are in #3044; reopen there if the roadmap changes.
